vault/request_forwarding_service.proto

```diff
@@ -20,6 +20,9 @@ message EchoRequest {
 	NodeInformation node_info = 6;
 	uint64 raft_term = 7;
 	string raft_desired_suffrage = 8;
+	string raft_upgrade_version = 9;
+	string raft_redundancy_zone = 10;
+	string sdk_version = 11;
```

Comment:
I'd rather not enshrine the term "sdk version" in anything that'll be hard to change, since I hope to move the version out of the sdk.  I guess this is ok since protobuf lets us rename fields right?

Reply:
That's my understanding, although my intention was to use sdk_version to denote the version of Vault in use, rather than the fact that it's set in the SDK. It felt unambiguous to me. Is there a term you'd prefer?

Reply:
Haha if I had a better term I would've suggested it.  It's fine for now so long as it's not exposed anywhere that would lead to a backwards compat concern if we wanted to change it.